modules/jaeger-rn-fixed-issues.adoc

```diff
@@ -15,10 +15,4 @@ Fix - What did we change to fix the problem?
 Result - How has the behavior changed as a result?  Try to avoid “It is fixed” or “The issue is resolved” or “The error no longer presents”.
 ////
 
-BZ# https://bugzilla.redhat.com/show_bug.cgi?id=000001[000001]
-
-There was a werewolf in the last release.  You would have only encountered this werewolf if it was a full moon and you were outside.  The werewolf has been shot with a silver bullet and no longer exists.
-
-BZ# https://bugzilla.redhat.com/show_bug.cgi?id=000005[000005]
-
-There was an unrecognized vampire in the last release.  As a result, the vampire bit a lot of people and sired multiple vampires.  But after the liberal application of sunlight, garlic, and holy water, all vampires have been removed from the current release and you no longer need to wear turtlenecks when going outside.
+* link:https://issues.jboss.org/browse/TRACING-1631[TRACING-1631] Multiple Jaeger production instances causing Elasticsearch certificate issue.  When multiple service meshes were installed, all of the Jaeger Elasticsearch instances had the same Elasticsearch secret instead of individual secrets, which prevented the Elasticsearch Operator from communicating with all of the meshes.
```

Comment:
"Multiple Jaeger production instances, using same name but within different namespaces, causing..."

Reply:
"from communicating with all of the meshes" -> "from communicating with all of the Elasticsearch clusters"
Is service mesh relevant here? Maybe could mention it at the end, as an example - e.g. "This issue impacted Service Mesh"?

Reply:
I thought this issue came up as part of Service Mesh?